src/main/java/com/qaprosoft/carina/demo/mobile/gui/mfp/pages/android/LoginPage.java

```diff
@@ -135,7 +135,7 @@ public DashboardPageBase login(String email, String password) {
         HomeScreenModalBase homeScreenModal = initPage(getDriver(), HomeScreenModalBase.class);
         homeScreenModal.closeModal();
         DashboardPageBase dashboardPage = initPage(getDriver(), DashboardPageBase.class);
-        Assert.assertTrue(dashboardPage.isPageOpened(), "Login fail, Home page isn't open");
+        Assert.assertTrue(dashboardPage.isPageOpened(), "Login fail, Dashboard page isn't open");
```

Comment:
small remark - Login failed, - we write our messages in past form: page isn't opened, modal isn't shown.